### PR TITLE
fix the KLC S4.2 for all symbols

### PR DESCRIPTION
**Pull request by dreamcmi**
I tried to fix KLC S4.2 for all current symbols, but may be still has some errors, please review.

https://github.com/espressif/kicad-libraries/issues/71

### CODE REVIEW

**Comment by pedrominatel** [comment]
Hi, @dreamcmi. Thank you for your contribution. I'll take a look ASAP.

**Comment by dreamcmi** [comment]
Hi @pedrominatel, I have tested the library with KiCad 7 release.

**Comment by pedrominatel** [comment]
Hi @dreamcmi 

Can you fix the conflicts?

**Comment by dreamcmi** [comment]
Hi @pedrominatel 
I have fixed the conflicts, please review.

File: libraries/Espressif.kicad_sym [hunk 1]
```diff
@@ -1269,7 +1274,7 @@
         (name "GPIO23" (effects (font (size 1.27 1.27))))
         (number "21" (effects (font (size 1.27 1.27))))
       )
-      (pin no_connect line (at -27.94 -12.7 0) (length 2.54)
```
Comment (pedrominatel) [inline libraries/Espressif.kicad_sym:1272]: Same here

**Review by pedrominatel — COMMENTED** [review]
Some NC pins must be hidden.

**Comment by dreamcmi** [comment]
Done, please review.

**Pull request merged**